docs/reference/islandora_defaults_reference.md

```diff
@@ -0,0 +1,391 @@
+# Drupal Configuration Provided By Islandora Defaults
+
+This is a list of the configuration items provided by the Islandora Defaults module. 
+
+## Entities
+
```

Comment:
consider adding link to Glossary:
``(see Glossary: [Entity](../user-documentation/glossary.md#entity))``